pros/src/lcd/macros.rs

```diff
@@ -7,23 +7,26 @@ use core::fmt::{self, Write};
 use super::WRITER;
 
 #[doc(hidden)]
-pub fn _print(args: fmt::Arguments) {
+pub fn _llemu_print(args: fmt::Arguments) {
     WRITER.lock().write_fmt(args).unwrap();
 }
 
 #[macro_export]
-macro_rules! print {
+macro_rules! llemu_print {
```

Comment:
To be honest, does this really deserve its own macro rather than just using `write!`/`writeln!` and exposing the llemu writer?

Reply:
It's more convenient to have a macro. That being said, its not a bad idea to also expose the writer.

Reply:
Keeping macros around for the time being (whether or not they should be there in the long run is a valid discussion). Writer can be exposed in a separate PR.